eksupgrade/src/preflight_module.py

```diff
@@ -11,11 +11,28 @@
 
 from eksupgrade.utils import get_package_dict
 
-urllib3.disable_warnings(urllib3.exceptions.InsecureRequestWarning)
 from .k8s_client import get_default_version, loading_config
 
+urllib3.disable_warnings(urllib3.exceptions.InsecureRequestWarning)
+
 logger = logging.getLogger(__name__)
 
+CLUSTER_ROLES: List[str] = [
```

Comment:
we can remove these and the ones below - if they are removed they will be resolved/replaced by EKS and/or notified via the cluster health findings. roles are added/removed as needed to facilitate the automation of EKS